index.html

```diff
@@ -3093,6 +3097,46 @@ <h4>Other considerations</h4>
 <p>This helps ensure that we represent the worldwide community of users with our technologies, and makes the specification appear more relevant to the global user.</p>
 
 <p>Try to choose names that represent people from each major region around the world, rather than just a handful of European names. Also, try to have a balance between male and female names.</p>
+
+<aside class="note">
+	<p>Many specifications provide examples, such as user stories or use cases, that use people's names as a means of enhancing the narrative. Some groups even have practices, such as security specifications using the names "Alice" and "Bob", to provide a certain level of consistency. Inclusiveness should be an important goal when building systems and services, hence the suggestion to use globally diverse names in forming examples.</p>
+</aside>
+
+<p>No list of names can be completely agnostic in dealing with cultural and gender-related issues. To assist specification writers in creating more inclusive examples, this document provides a list of names drawn from across many cultures. The names are paired with generally-feminine names followed by generally-masculine names. Note well that many names are gender indeterminate. When choosing items from this list (or selecting your own names to use in examples) try to avoid selecting only those from specific regions or cultures. Choosing names with non-ASCII characters may help remind implementers of the need for Unicode support.</p>
+
+<aside class="note">
+	<p>The list of names provided here are rendered in the Latin script because most specifications are written in English. When names are taken from non-Latin-script languages or cultures, sometimes the non-Latin representation is also provided as a reminder that names are in no way limited to the Latin script.</p>
```

Comment:
```suggestion
	<p>The list of names provided here is rendered in the Latin script because most specifications are written in English. When names are taken from non-Latin-script languages or cultures, sometimes the non-Latin representation is also provided as a reminder that names are in no way limited to the Latin script.</p>
```